scripts/constants.js

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
Let's call this `config.js` instead of `constants.js`. I suspect this file will undergo some refactoring after we start adding on to the goal route so I won't comment on the structure of it right now.